docs/aws: update AWS deploy docs after testing (#491)
pingcap/tidb-operator: deploy/aws/README.md

* 2 c4.4xlarge instances for TiDB
* 1 c5.xlarge instance for monitor

> You can change default values in `variables.tf` (like cluster name and versions) as needed. The default `cluster_name` is `my-cluster`.

Reviewer: ```suggestion
You can change default values in `variables.tf` (like the cluster name and versions) as needed. The default value of `cluster_name` is `my-cluster`.
```

``` shell
$ git clone --depth=1 https://github.com/pingcap/tidb-operator
$ cd tidb-operator/deploy/aws
$ terraform init
$ terraform apply
```

> It might take 10 minutes or more for the process to finish. After `terraform apply` is executed successfully, some basic information will be printed to the console, you can access the `monitor_endpoint` using your web browser.

Reviewer: ```suggestion
It might take 10 minutes or more for the process to finish. After `terraform apply` is executed successfully, some basic information is printed to the console. You can access the `monitor_endpoint` using your web browser.
```

> > **Note:** You can use `terraform output` command to get those information again.

Reviewer: ```suggestion
> **Note:** You can use the `terraform output` command to get that information again.
```

helm ls
```

# Destory

It may take some while to finish destroying the cluster.

```shell
$ terraform destroy
```

> > **NOTE:** You have to manually delete the EBS volumes in AWS console after running `terraform destroy` if you don't need the data on the volumes any more.

Reviewer: ```suggestion
> **Note:** You have to manually delete the EBS volumes in AWS console after running `terraform destroy` if you do not need the data on the volumes anymore.
```
